docs/fixes/2026-08-20-windows-go-test-unlinkat-retry.md

```
@@ -0,0 +1,59 @@
+# Fix: retry the transient Windows `go: unlinkat ...` race in acceptance test shards
+
+**Date:** 2026-08-20
+
+## Summary
+
+`Acceptance Tests (windows, shard 10/10)` (GitHub Job ID: 96582175875) failed even though every
+one of its 39 packages reported `ok`. The actual failure was Go's own toolchain diagnostic:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)